python/aws_lambda_powertools/metrics/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class MetricUnitError(Exception):
+    pass
+
+
+class SchemaValidationError(Exception):
+    pass
+
+
+class MetricValueError(Exception):
+    pass
+
+
+class UniqueNamespaceError(Exception):
+    pass
```

Comment:
Document this file of when these will be raised

Reply:
By convention we document where it's raised (`base.py`) under `Raises` doc